src/main/java/uk/ac/cam/cl/dtg/isaac/dto/IsaacWildcardDTO.java

```diff
@@ -29,7 +29,7 @@
 /**
  * ********************************************
  * <br>
- * Note: This class is currently not used.!! TODO: make sure mapping is completed.
+ * Note: This class is currently not used.!!
```

Comment:
Is this comment actually correct? It seems to be used by `IsaacWildcard`. If it's being used, I think we can get rid of this whole comment block.

Reply:
The _only_ place I can find where this class is referenced is an annotation `@DTOMapping(IsaacWildcardDTO.class)` in the non-DTO version. I assume we still want to keep this around but the comment appears to be accurate as far as I can see?